packages/widget-todo/src/Context.tsx

```diff
@@ -20,6 +21,7 @@ const TodoProvider = ({ children }: { children: React.ReactElement }) => {
 
   const [showAddDialog, setShowAddDialog] = useState(false)
   const [showEditDialog, setShowEditDialog] = useState<string | undefined>()
+  const [showCloudSyncFeature, setShowCloudSyncFeature] = useState(false)
```

Comment:
Given we only use these values in one single React component there is no need to put it into the context, just define it as part of the component where these variables are being used. The context is really to allow many components re-use the same states

Reply:
Is this also applicable to widget-todo and widget-notes?

Reply:
How important is listening to its event ? 
```

```

> @lorenzejay where it says `Clipboard Snippets` can we just say snippets please? We'll handle the renaming as a consolidated effort in a separate ticket. Thanks!
> 
> @lorenzejay @christian-bromann we should probably have a "Thanks for your feedback" message when users hit yes/no. Maybe with a call to action to "Join Discord"?

WFM. I think after click on yes/no -> feedback text = "Thank you for your answer. Join our discord channel if you like to stay updated with upcoming features or if you have other feature requests.